grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -210,13 +211,21 @@ class StockMovementController {
             if (stockMovement?.shipment?.currentStatus == ShipmentStatusCode.PENDING || !stockMovement?.shipment?.currentStatus) {
                 try {
                     stockMovementService.deleteStockMovement(params.id)
-                    flash.message = "Successfully deleted stock movement with ID ${params.id}"
+                    params.notificationMessage = g.message(
```

Comment:
params.notificationMessage doesn't seem necessary. Just use the flash.message. Objects placed in flash scope should persist through a redirect. 

https://docs.grails.org/3.1.1/ref/Controllers/flash.html